src/main/java/pageobjectsnew/AdminPageNew.java

```diff
@@ -0,0 +1,91 @@
+package pageobjectsnew;
+
+import businessobjects.User;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.LoadableComponent;
+import utility.Constants;
+
+import static org.testng.Assert.assertTrue;
+import static utility.Services.ManageUrlService.getCurrentURL;
+import static utility.Services.WaiterService.waitForElementVisible;
+import static utility.Services.WaiterService.waitPageLoader;
+import static utility.Services.WebElementService.clickOnElement;
+import static utility.Services.WebElementService.sendKeysClear;
+
+/**
+ * Created by user on 14.05.17.
+ */
+public class AdminPageNew extends LoadableComponent<AdminPageNew> {
+
+    protected WebDriver driver;
+
+    public AdminPageNew(WebDriver driver) {
+        this.driver = driver;
+        PageFactory.initElements(driver, this);
+    }
+
+
+    @FindBy(name = "login")
+    public WebElement email;
+
+    public WebElement password;
+
+    @FindBy(className = "ui-dialog-titlebar-close")
+    public WebElement closeTrialPopUpCross;
+
+    public WebElement leftMenu;
+
+    @FindBy(css = "[type=submit]")
+    public WebElement logInButtonCss;
+
+    public AdminPageNew login(User user, String method){
+        sendKeysClear(email, "Email form", user.getEmail(), driver);
+
+        sendKeysClear(password, "Password form", user.getPassword(), driver);
+
+        switch (method){
```

Comment:
It will coll to create enum for this action.

In more complicated situations, if the behavior is different, but action named the same based on some conditions you should use Strategy pattern  - for example ride a transport should be different for bike, car, bus, horse.